contracts/v1/ergonames_v1_collection.es

```diff
@@ -37,8 +39,9 @@
 
     // ===== Relevant Variables ===== //
     val minerFeeErgoTreeHash: Coll[Byte] = fromBase16("e540cceffd3b8dd0f401193576cc413467039695969427df94454193dddfb375")
-    val ergonameCollectionTokenId: Coll[Byte] = SELF.tokens(1)._1
-    val ergonameCollectionTokenAmount: Long = SELF.tokens(1)._2
+    val ergonameCollectionTokenAmount = SELF.tokens.fold(0L, { (sum: Long, t: (Coll[Byte], Long)) =>
```

Comment:
Could just use filter?

Reply:
Or is it possible for same token to be in different sections of the tokens collection?